examples/c-api/buildsystem/main.c

```diff
@@ -64,11 +64,18 @@ static bool fs_get_file_contents(void* context, const char* path,
   printf(" -- read file contents: %s\n", path);
   fflush(stdout);
 
+#if defined(_WIN32)
+  FILE* fp;
+  if (fopen_s(&fp, path, "rb")) {
```

Comment:
`_wfopen_s`, please :)

Probably not a bad idea to move this into PlatformUtility...

Reply:
This one's in the client C sample code. I assume that it shouldn't be using the C++ functions in PlatformUtility? _wfopen_s makes sense though.

Reply:
Oh, right. I wasn't looking at the path. But yes, still should use _wfopen_s.